src/utils/helpers/helpers.js

```diff
@@ -15,4 +15,10 @@ export class Cookie{
   static del(cname){
       document.cookie = `${cname}=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;`;
   }
+}
+
+export class CommonHelpers {
+  static capitalizeTheme = (str) => str.toLowerCase()
+    .split(/\s+/)
+    .map((word) => word[0].toUpperCase() + word.substring(1)).join(' ');
```

Comment:
better to move this class in separate file CommonHelpers.js and rename helpers.js into Cookie.js

Reply:
done